src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -163,6 +138,31 @@ using PIC_Ions = Particles<
     ParticleFlagsIons
 >;
 
+/*--------------------------- electrons --------------------------------------*/
+
+/* ratio relative to BASE_CHARGE and BASE_MASS */
+value_identifier( float_X, MassRatioElectrons, 1.0 );
+value_identifier( float_X, ChargeRatioElectrons, 1.0 );
+
+using ParticleFlagsElectrons = bmpl::vector<
+    particlePusher< UsedParticlePusher >,
+    shape< UsedParticleShape >,
+    interpolation< UsedField2Particle >,
+    current< UsedParticleCurrentSolver >,
+    massRatio< MassRatioElectrons >,
+    chargeRatio< ChargeRatioElectrons >
+#if( ENABLE_SYNCHROTRON_PHOTONS == 1 )
```

Comment:
please remove `ENABLE_SYNCHROTRON_PHOTONS` and please it in the example without the guard.